### ISSUE TITLE
Heads up!

### ISSUE THREAD

**Issue by ItsCEED** [dajk/hltv-api#5]
https://twitter.com/hltvorg/status/927597650320416768

**Comment by dajk** [comment]
Yes, agree. I think would be nice to have live streaming data throughout the module..
Definitely I should be working more heavily on this api.
